### PR TITLE
[Remove Stream/Optional usage] LPS-170484 document-library

### PR DESCRIPTION
**Pull request by JanisZhang**
This PR is to Remove usages of Stream/Optional on Method Return Value for module: `document-library`. Thanks!

### CODE REVIEW

**Comment by dantewang** [comment]
@JanisZhang Please include the module you change in PR title, thanks.

File: modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/servlet/filter/DLVideoEmbedFilter.java [hunk 1]
```diff
@@ -139,12 +138,15 @@ private FileEntry _getFileEntry(
 		throws PortalException {
 
 		if (_PATH_SEPARATOR_FILE_ENTRY.equals(pathParts.get(1))) {
-			Optional<FileEntry> fileEntryOptional = _resolveFileEntry(
+			FileEntry fileEntry = _resolveFileEntry(
```
Comment (dantewang) [inline modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/servlet/filter/DLVideoEmbedFilter.java:141]: @JanisZhang I think this commit should belong to (be merged into) the last commit?

File: modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/display/context/DLItemSelectorViewDisplayContext.java [hunk 2]
```diff
@@ -351,18 +354,18 @@ public void setShowDragAndDropZone(boolean showDragAndDropZone) {
 		_showDragAndDropZone = showDragAndDropZone;
 	}
 
-	private Optional<Long> _getFileEntryTypeIdOptional() {
+	private Long _getFileEntryTypeIdOptional() {
```
Comment (dantewang) [inline modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/display/context/DLItemSelectorViewDisplayContext.java:357]: @JanisZhang If the method doesn't return an `Optional<>`, the method name should no longer contain "Optional".

**Review by dantewang — CHANGES_REQUESTED** [review]
@JanisZhang Some minor issues, please see my comments and update.

cc @tinatian

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Review by dantewang — APPROVED** [review]

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by JanisZhang** [comment]
no conflicts (12/09 9:35 am)

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/127497

**Pull request closed without merging**